src/datepicker/datepicker-input.ts

```diff
@@ -204,6 +204,14 @@ export class NgbInputDatepicker implements OnChanges,
    */
   @Input() container: string;
 
+  /**
+   * A selector or html element specifying the element the datepicker popup should be positioned by.
```

Comment:
selector → css selector ? (if you remove the second redundant line)
by → against ?